src/past-purchase/index.html

```diff
@@ -0,0 +1,11 @@
+<yield>
+  <gb-link if="{ $pastPurchase.value && this.state.pastPurchases > 0 }" class="gb-autocomplete-target" data-past-purchase onmouseenter="{ $autocomplete.onHover }" onclick="{ this.state.onClick }">
```

Comment:
`this` should not show in html code, you should be able to use your alias here and it equals `state`, so if your alias is `$pastPurchase`, then change `this.state.pastPurchases` to `$pastPurchase.pastPurchases`